src/components/selections/attributes.js

```diff
@@ -53,7 +53,28 @@ module.exports = overrideAll(templatedArray('selection', {
         valType: 'any',
         description: 'Sets the selection\'s end y position.'
     },
-
+    x_shift: {
+        valType: 'number',
+        dflt: 0,
+        min: -0.5,
+        max: 0.5,
+        editType: 'calc',
+        description: [
+            'Only relevant if xref is a (multi-)category axes. Shifts x0 and x1 by a fraction of',
+            'the reference unit.'
```

Comment:
What happens in the case of `axis.type: 'date'`?

Reply:
> Shifts x0 and x1

What if you wanted to shift x0 and x1 differently? Like to surround a particular bar with a rect?